src/System.Xml.XPath.XmlDocument/tests/System.Xml.XPath.XmlDocument.Tests.csproj

```diff
@@ -8,7 +8,7 @@
     <ProjectReference Include="..\src\System.Xml.XPath.XmlDocument.csproj" />
   </ItemGroup>
 
-  <ItemGroup Condition="'$(TargetFramework)' == '$(NetFrameworkMinimum)'">
+  <ItemGroup Condition="'$(TargetFrameworkIdentifier)' == '.NETFramework'">
```

Comment:
OK I understand this one, as it is grouping all the netfx versions, not just the minimum one. It's not as clear with the .NETCoreApp case.